src/screens/CreatePost/index.js

```diff
@@ -117,8 +116,6 @@ const CreatePost = () => {
   const [isUploadingPhotoCamera, setIsUploadingPhotoCamera] = React.useState(false);
 
   const {setHashtags} = useHastagMention('');
-  const [client] = React.useContext(Context).client;
-  const [user] = React.useContext(Context).profile;
   const [allTaggingUser, setAllTaggingUser] = React.useState([]);
   const animatedReminder = React.useRef(new Animated.Value(0)).current;
   const debounced = React.useCallback(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like you've removed the usage of `Context` for `client` and `profile`. If these values are not used anymore, it's a good change. However, if they're still needed somewhere in your component, you might want to consider passing them as props or using a custom hook to fetch them.



<!-- This is an auto-generated comment by OSS CodeRabbit -->